tools/noopfactory/build.gradle.kts

```diff
@@ -24,7 +24,7 @@ plugins {
 dependencies {
     implementation(Dependencies.Libraries.Kotlin)
     implementation(Dependencies.Libraries.KotlinReflect)
-    implementation("com.squareup:kotlinpoet:1.5.0")
+    implementation("com.squareup:kotlinpoet:1.7.2")
```

Comment:
update is needed because `1.5.0` strictly depends on `kotlin-reflect:1.3.50` (previous language level)